### PR TITLE
Recover advocates from airtable

### LINKED ISSUE #1071: Recover information from Airtable
Size: M

### PR DESCRIPTION
**Pull request by y4izus**
Fixes #1071

### CODE REVIEW

File: store/modules/advocates.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+  },
+  mutations: {
+    setAdvocates (state: any, payload: any) {
+      state.advocates = payload
+    }
+  },
+  actions: {
+    async fetchAdvocates () {
+      const eventsModule = await import('~/content/advocates/advocates.json')
```
Comment (techtolentino) [inline store/modules/advocates.ts:56]: should this be `advocatesModule` ?

File: components/advocates/MeetTheAdvocates.vue [hunk 2]
```diff
@@ -62,3 +62,12 @@ export default class extends Vue {
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
```
Comment (techtolentino) [inline components/advocates/MeetTheAdvocates.vue:67]: I don't think this is being used

**Review by techtolentino — COMMENTED** [review]
Hey @y4izus - this is great!

I just had a couple of questions


thank you

File: store/modules/advocates.ts (hunk 1 above)
Comment (y4izus) [reply store/modules/advocates.ts:56]: Yes!! thank you!

**Review by korgan00 — APPROVED** [review]

File: components/advocates/MeetTheAdvocates.vue (hunk 2 above)
Comment (y4izus) [reply components/advocates/MeetTheAdvocates.vue:72]: I think is needed for the `copy_title` and `copy_paragraph`

**Review by korgan00 — APPROVED** [review]

**Pull request merged**